godal.go

```diff
@@ -2627,13 +2674,117 @@ func (f *Feature) SetGeometry(geom *Geometry, opts ...SetGeometryOption) error {
 	return cgc.close()
 }
 
+//SetGeometryColumnName set the name of feature first geometry field.
+func (f *Feature) SetGeometryColumnName(name string) {
```

Comment:
TODO: return error for incorrect field name

Reply:
What do you consider as incorrect ? I tried several values (special characters, existing field name...) and OGR_GFld_SetName never raise an error or a warning.

Reply:
ok, the error would arise at CreateFeature() time